README.md

```diff
@@ -328,6 +328,35 @@ to your chosen fungible token for now. Then, use that public path for your royal
 and in the future, you will be able to easily update the link at that path to use the
 fungible token switchboard instead.
 
+## Contract metadata
+
+Now that contract borrowing is released, you can also implement the [Resolver](./contracts/Resolver.cdc) interface on your contract
+and resolve views from there. As an example, you might want to allow your contract to resolve NFTCollectionData and NFTCollectionDisplay
+so that platforms do not need to find an NFT that belongs to your contract to get information about how to set up or show your collection
```

Comment:
```suggestion
so that platforms do not need to find an NFT that belongs to your contract to get information about how to set up or show your collection.
```